core/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -298,6 +305,7 @@ public NodeIndicesStats stats(boolean includePrevious, CommonStatsFlags flags) {
     /**
      * Returns <tt>true</tt> if changes (adding / removing) indices, shards and so on are allowed.
      */
+    @Override
     public boolean changesAllowed() {
```

Comment:
I actually think we can get rid of this entirely. We start the `IndicesService` before `IndicesClusterStateService` and stop it in the reverse order in `Node.java`. I think we can just rely on the state of `IndicesClusterStateService` and don't add this to the interface at all.